docs/articles/dockerfile_best-practices.md

```diff
@@ -146,16 +146,33 @@ with backslashes.
 Probably the most common use-case for `RUN` is an application of `apt-get`.
 When using `apt-get`, here are a few things to keep in mind:
 
-* Don’t do `RUN apt-get update` on a single line. This will cause
-caching issues if the referenced archive gets updated, which will make your
-subsequent `apt-get install` fail without comment.
-
 * Avoid `RUN apt-get upgrade` or `dist-upgrade`, since many of the “essential”
```

Comment:
Looks good! Spotted a tiny typo while reading (haven't given it a thorough read);

> installs the latest package versions without no further coding or manual

"without no" -> "with no"

Should we add a "post install cleanup" to the "full" example? This is what my standard blocks look like;

```Dockerfile
RUN apt-get update && apt-get install -y \
    foo \
    bar \
    baz \
 && apt-get clean \
 && rm -r /var/lib/apt/lists/*
```